source/ide/wiki/wiki_methods.bas

```diff
@@ -6,27 +6,29 @@ FUNCTION Back2BackName$ (a$)
     Back2BackName$ = a$
 END FUNCTION
 
-FUNCTION Wiki$ (PageName$)
+FUNCTION Wiki$ (PageName$) 'Read cached wiki page (download, if not yet cached)
     Help_PageLoaded$ = PageName$
-    PageName2$ = PageName$
 
-    DO WHILE INSTR(PageName2$, " ")
-        ASC(PageName2$, INSTR(PageName2$, " ")) = 95
-    LOOP
-    DO WHILE INSTR(PageName2$, "&")
-        i = INSTR(PageName2$, "&")
-        PageName2$ = LEFT$(PageName2$, i - 1) + "%26" + RIGHT$(PageName2$, LEN(PageName2$) - i)
-    LOOP
-    DO WHILE INSTR(PageName2$, "/")
-        i = INSTR(PageName2$, "/")
-        PageName2$ = LEFT$(PageName2$, i - 1) + "%2F" + RIGHT$(PageName2$, LEN(PageName2$) - i)
-    LOOP
+    'Escape all invalid and other critical chars in filenames
+    PageName2$ = ""
+    FOR i = 1 TO LEN(PageName$)

```

Comment:
This seems like something that could be thrown in a `UrlExpandSpecialCharacters$()` function, or something along those lines :)